docs/.env.example

```diff
@@ -0,0 +1,3 @@
+# Create one with no scope selected on https://github.com/settings/tokens/new
+# This token is used for fetching the repository releases.
+GITHUB_TOKEN=
```

Comment:
I believe we don't need a token for fetching only content using an unstorage source. It is only needed for listing operations.